client/src/modules/Opportunities/components/PublicLink/index.test.tsx

```diff
@@ -42,8 +42,7 @@ describe('PublicLink', () => {
 
     expect(mockCopyToClipboard).toHaveBeenCalledWith(mockUrl);
 
-    const notification = screen.getByText('Copied to clipboard');
-
-    expect(notification).toBeInTheDocument();
+    // const notification = screen.getByText('Copied to clipboard');
+    // expect(notification).toBeInTheDocument();
```

Comment:
plus `async` at line 36 
```suggestion
   const notification = await screen.findByText('Copied to clipboard');
   expect(notification).toBeInTheDocument();
```